app/src/main/java/org/mozilla/firefox/vpn/user/data/UserRepository.kt

```diff
@@ -23,7 +23,7 @@ class UserRepository(
     suspend fun getLoginInfo(): Result<LoginInfo> {
         return try {
             val loginQueryBuilder = LoginQueryBuilder(referralManager.getUserReferral())
```

Comment:
Do we need to add a parameter name for the referral URL since LoginQueryBuilder looks like a general class for any type of query parameter instead of referrer-only? For example, `"referrer=${referralManager.getUserReferral()}"`?

Reply:
we don't need it. the value return from `referralManager.getUserReferral()}` will look like `utm_source=google&utm_campaign=organic....` so we don't need a parameter here.